src/Microsoft.DocAsCode.Build.TableOfContents/TocResolver.cs

```diff
@@ -59,56 +104,91 @@ private TocItemInfo ResolveItemCore(TocItemInfo wrapper, Stack<FileAndType> stac
                         {
                             item.Items[i] = ResolveItem(new TocItemInfo(file, item.Items[i]), stack).Content;
                         }
-                        if (string.IsNullOrEmpty(item.Homepage) && string.IsNullOrEmpty(item.HomepageUid))
+                        if (string.IsNullOrEmpty(item.TopicHref) && string.IsNullOrEmpty(item.TopicUid))
                         {
                             var defaultItem = GetDefaultHomepageItem(item);
                             if (defaultItem != null)
                             {
-                                item.Homepage = defaultItem.Href;
-                                item.HomepageUid = defaultItem.Uid;
+                                item.TopicHref = defaultItem.Href;
+                                item.TopicUid = defaultItem.TopicUid;
                             }
                         }
```

Comment:
It's useful when getting the default homepage for a toc. To avoid confusion, use AggregatedHref and AggregatedUid instead.

---
In reply to: [73487131](https://github.com/dotnet/docfx/pull/537#discussion_r73487131) [](ancestors = 73487131)